docs/user/BYOnetwork.md

```diff
@@ -0,0 +1,23 @@
+## Supported scenarios and requirements for using existing AWS network resources
+
+The table below shows the scenarios supported with using existing/bring your own(BYO) network resources:
+
+| <div style="width:70px">Scenario</div> | <div style="width:70px">BYO resource</div> | <div style="width:500px">Requirements</div> | <div style="width:100px">Resources that will be created</dvi>|
+| :--- | :--- | :--- | :--- |
+| 1 | VPC | <ul><li>VPC block size must be IPv4 CIDR block with '/16' netmask (supports 65,536 IP addresses)</li><li>`DNS hostnames` and `DNS resolution` are enabled</li><li>`var.vpc_id` must be provided</li><li>`var.subnets` CIDR blocks must match with VPC CIDR IPv4 CIDR block</li></ul> | Subnets (NAT_id, routes and Route Table association) and Security Group|
```

Comment:
Using the var.<variable_name) format isn't used anywhere else in our readme's. We always use the variable names as documented in CONFIG-VARS.md in variables.tf. So we have no context to this format. So from example replace var.vpc_id with vpc_id and var.subnets with subnet_ids. This same observation applies to all var.* formatted variables in this document.